example/fake-provider-cf.yml

```diff
@@ -39,7 +53,10 @@ Resources:
             Statement:
               - Effect: Allow
                 Action: "S3:PutObject"
-                Resource: !Sub "arn:aws:s3:::${Bucket}/fake-provider-*"
+                Resource: !Sub "arn:aws:s3:::${Bucket}/fake-*"
+              - Effect: Allow
+                Action: "S3:PutObject"
+                Resource: !Sub "arn:aws:s3:::${S3ProviderBucket}/*"
```

Comment:
It may become apparent as I go further through this - but why are both buckets needed? Writes/perms the proc lambda role?

Reply:
And .... there it is.  Disregard.